recipe/meta.yaml

```diff
@@ -7,9 +7,12 @@ package:
 source:
     url: https://bitbucket.org/chchrsc/kealib/downloads/kealib-{{ version }}.tar.gz
     sha256: 1c80489f17114a229097c2e8c61d5e4c82ea63ae631c81a817fef95cfd527174
+    patches:
+      # by default kealib forces C++98 mode which breaks other things built in C++11 mode
+      - disablecpp98.patch
 
 build:
-    number: 0
+    number: 1
```

Comment:
I guess it would be safer to remove the build number `0` to avoid conda picking it up.
I'll do that.